### PR TITLE
adding category filtering

### PR DESCRIPTION
**Pull request by kzzztkt**

### CODE REVIEW

File: src/components/Nav.jsx [hunk 1]
```diff
@@ -1,22 +1,48 @@
...
+
+  function handleSearch(event){
+    event.preventDefault();
+    if(event.target.categories.value === 'all'){
+      navigate(`/reviews`)
+    } else {
+    navigate(`/reviews/categories/${event.target.categories.value}`)
+    }
+  }
+
```
Comment (elijaharley) [inline src/components/Nav.jsx:19]: Really nice use of conditional logic to sort here.

File: src/components/Reviews.jsx [hunk 2]
```diff
@@ -49,6 +64,7 @@ function Reviews() {
         </div>
         </Link>
             )
+            }
```
Comment (elijaharley) [inline src/components/Reviews.jsx:67]: I like that you've made the whole card a link here, and the hover CSS here is really effective.

**Review by elijaharley — APPROVED** [review]
Really nice work on this ticket, your code is clean and readable and you've made a really good job of the CSS

**Comment by kzzztkt** [comment]
> Really nice work on this ticket, your code is clean and readable and you've made a really good job of the CSS

Thank you @elijaharley :)

**Pull request merged**